lib/services/plugins-service.ts

```diff
@@ -194,7 +206,21 @@ export class PluginsService implements IPluginsService {
 		}).future<void>()();
 	}
 
-	private getInstalledPluginByName(pluginName: string): IPlugin[] {
+	public getPluginBasicInformation(pluginName: string): IBasicPluginInformation {
+		let result: IBasicPluginInformation = Object.create({});
+		let pluginParts = pluginName.split("@");
+		result.name = pluginParts[0];
+		result.version = pluginParts[1];
```

Comment:
You can use [destructuring](https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#destructuring-in-declarations-and-assignments) and collapse the 5 lines into 1.